app/api.py

```diff
@@ -791,3 +798,21 @@ def get_element_text(self, doc, tag_name):
             return text.encode('utf-8')
         else:
             return None
+
+    #Send hit to GA
+    def gamp(e,ec):
+        ga_cid = uuid.uuid4()
+        params = urllib.urlencode({
+            'v': 1,
+            'tid': 'UA-17770602-2',
```

Comment:
Use self.env.analytics_id instead of 'UA-17770602-2' here. We make Analytics ID configurable via config. To test it in your local environment, you can set config "analytics_id" by following instruction here:
https://github.com/google/personfinder/wiki/DeveloperFaq#how-to-readwrite-config-values

You should also handle a case where self.env.analytics_id is None. In that case, you should not send Measurement Protocol.